test/e2e/admin_portal.go

```diff
@@ -134,6 +134,86 @@ var _ = Describe("Admin Portal E2E Testing", func() {
 		}
 	})
 
+	It("Should be able to populate feature flag panel correctly", func() {
+		const (
+			cluster_feature_headings = 24
```

Comment:
Same goes for this one, `clusterFeatureHeadings`.

Reply:
Yup!

Reply:
Done!